navigation/BottomTabNavigator.tsx

```diff
@@ -119,6 +120,18 @@ function SettingsNavigator() {
           headerTitleStyle: styles.headerTitle,
         }}
       />
+      <SettingsStack.Screen
+        name="AboutScreen"
+        component={AboutPage}
+        options={{
+          headerBackTestID: 'headerBack',
+          headerTintColor: Colors.light.white,
+          headerBackTitle: 'Back',
```

Comment:
I may tweak this a bit to not include the word "Back" post-merge as we didn't include it in other screens but looks good!

Reply:
Aye no worries. I did actually copy the convention used here https://github.com/heylinda/heylinda-app/blob/829992eb210493823712c3eb51fced884ec892b9/navigation/BottomTabNavigator.tsx#L82 

But yea that sounds good 👍